src/queue-list/laboratory-patient-list.component.tsx

```diff
@@ -187,6 +207,31 @@ const LaboratoryPatientList: React.FC<LaboratoryPatientListProps> = () => {
                 }}
               >
                 <TableToolbarContent>
+                  <Layer style={{ margin: "5px" }}>
+                    <Dropdown
+                      id="orderStatus"
+                      initialSelectedItem={"All"}
+                      label=""
+                      titleText={
+                        t(
+                          "filter_orders_by_status",
+                          "Filter Orders by status"
+                        ) + ":"
+                      }
+                      type="inline"
+                      items={[
+                        "All",
+                        "RECEIVED",
+                        "IN_PROGRESS",
+                        "COMPLETED",
+                        "EXCEPTION",
+                        "ON_HOLD",
+                        "DECLINED",
+                      ]}
```

Comment:
let's extract these to a helpers or constants file for easy reuse

Reply:
> let's extract these to a helpers or constants file for easy reuse

hello @pirupius i have made some changes including the ability to translate the statues for other implementations in different languages other than hard coding them in english. This brings me to think that since the statuses are not affecting the back-end code , we need not keep them as constants in a separate file since they are rendered in the UX workflow